### DISCUSSION TITLE
About Mongo DB

### DISCUSSION THREAD

**Discussion by ObaidKazi** [filamentphp/filament#6384] [General]
Can we use mongodb instead of mysql in filamint if yes then what sort of changes need to done?

**Comment by danharrin** [comment]
You could use an Eloquent MongoDB adapter, but I really suggest you don't. It's not officially supported, I don't use MongoDB, and things probably wont work correctly. MySQL or PostgreSQL are good enough for 99% of projects, you should just use one of those.

**Reply by hardiklakhalani** [reply]
> You could use an Eloquent MongoDB adapter, but I really suggest you don't. It's not officially supported, I don't use MongoDB, and things probably wont work correctly. MySQL or PostgreSQL are good enough for 99% of projects, you should just use one of those.

is this case still the same?
